src/sections/Proud-maintainers/index.js

```diff
@@ -0,0 +1,199 @@
+import React from "react";
+import { Link } from "gatsby";
+
+import { Container, Row, Col } from "../../reusecore/Layout";
+import SectionTitle from "../../reusecore/SectionTitle";
+import Button from "../../reusecore/Button";
+
+import Image1 from "../../assets/images/app/proudMaintainers/envoy.png";
```

Comment:
Don't create the directory inside `app`, keep it under `images/proudMaintainers` only.

Reply:
Updated!

Reply:
Fixed